.codeclimate.yml

```diff
@@ -0,0 +1,34 @@
+---
+engines:
+  brakeman:
+    enabled: true
+    exclude_fingerprints:
```

Comment:
they're mentioned in the internal security tickets, I actually haven't read through the docs